src/fireflyClient.ts

```diff
@@ -768,9 +769,10 @@ export class FireflyClient {
   adjustLeverage = async (
     symbol: MarketSymbol,
     leverage: number,
-    perpetualAddress?: address
+    perpetualAddress?: address,
+    parentAddress?:string
```

Comment:
instead of making `perpetualAddress` and `parentAddress` optional like this, change parameters of  the method to be dictionary type arguments. 

The way you have set the parameters, if I am a subaccount of the parent then the only way I can adjust my paren'ts leverage is if I call this method like this: `client.adjustLeverage (ETH, 2,  ETHPerpAddress, ParentAddress)` you see the problem here? I am forced into providing the `ETHPerpAddress` in order to provide my parent account's address. Instead of this if we do dict args then we can do this: `client.adjustLeverage({symbol: ETH, leverage: 1, parentAddress: Address})`

Reply:
Also instead of asking a sub-account to provide parent address in each call, why not we create a property `parent` in client, which by default is set to the account it self. But we can expose a method `setParentAccount` to allow a sub account to specify its parent's address and then in all methods we read parent address from that field.

Reply:
Updated the parameters of the function to be dict type.